src/app/_services/global.service.ts

```diff
@@ -1,16 +1,23 @@
 import { Injectable } from '@angular/core';
 import { HttpHeaders } from '@angular/common/http';
 
+let token = sessionStorage.getItem('Token');
```

Comment:
Token is a dynamic and re-definable object - it should not be retrieved or set here but rather get retrieved right before each time you use it.